src/app/view/components/listing-data-table/action-header/action-header.ts

```diff
@@ -15,7 +15,7 @@ export class ActionHeaderComponent extends BaseComponent {
     @Input() options: ActionHeaderOptions;
     public dynamicOverflow = 'visible';
 
-    constructor( messageService: MessageService, private confirmationService: ConfirmationService) {
+    constructor(messageService: MessageService, private confirmationService: ConfirmationService) {
```

Comment:
Because we use it in the html... in fact we have some issues about that, I'm reading about how to handler props and services that are use in the templates.